From 34f0d9868ade5bcc153d573d2fc18ac8a8245e86 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Sebastian=20Dr=C3=B6ge?= <sebastian@centricular.com>
Date: Sun, 18 Feb 2018 11:37:18 +0200
Subject: [PATCH] configure: Add configure check for gmodule-no-export-2.0

This is needed for all code using the g_module_*() API.
---
 configure.ac | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/configure.ac b/configure.ac
index 8f53d9b7e..367492abb 100644
--- a/configure.ac
+++ b/configure.ac
@@ -186,6 +186,9 @@ dnl GLib is required
 GLIB_REQ=2.40.0
 AG_GST_GLIB_CHECK([$GLIB_REQ])
 
+dnl Needed by plugins that use g_module_*() API
+PKG_CHECK_MODULES(GMODULE_NO_EXPORT, gmodule-no-export-2.0)
+
 dnl checks for gstreamer
 dnl uninstalled is selected preferentially -- see pkg-config(1)
 AG_GST_CHECK_GST($GST_API_VERSION, [$GST_REQ], yes)
-- 
2.16.1

